fuse/src/main/java/tachyon/fuse/TachyonFuse.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.fuse;
+
+import java.nio.file.Paths;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import tachyon.Constants;
+import tachyon.client.ClientContext;
+import tachyon.conf.TachyonConf;
+
+/**
+ * Main entry point to Tachyon-FUSE
+ */
+public final class TachyonFuse {
+  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
+  private static TachyonConf sTachyonConf;
+
+  public static void main(String[] args) {
+    sTachyonConf = ClientContext.getConf();
+    final TachyonFuseOptions opts = parseOptions(args);
+    if (opts == null) {
+      System.exit(1);
+    }
+
+    final TachyonFuseFs fs = new TachyonFuseFs(sTachyonConf, opts);
+    final List<String> fuseOpts = opts.getFuseOpts();
+    fuseOpts.add("-odirect_io");
```

Comment:
could you javadoc heree why we need add this to `fuseOpts` (I guess to enable direct_io? is this for performance?)